.gitignore

```diff
@@ -29,6 +29,7 @@ test.rb
 /src/serialize.c
 /src/token_type.c
 /src/**/*.o
+/test/snapshots/fixtures/
```

Comment:
Hmmm, I don't think there's anything in the `/test/snapshots/fixtures/` dir. Did you mean the `/test/snapshots/` dir?

Reply:
Seems generated by a mistake at #1054.